src/chat/summarizers/dockerContainers.ts

```diff
@@ -1,37 +1,77 @@
-import { MarkdownString } from "vscode";
+import { MarkdownString, workspace } from "vscode";
 import { ContainerInspectResponse } from "../../clients/docker";
+import { DEFAULT_KAFKA_IMAGE_REPO, DEFAULT_SCHEMA_REGISTRY_REPO } from "../../docker/constants";
+import { LOCAL_KAFKA_IMAGE, LOCAL_SCHEMA_REGISTRY_IMAGE } from "../../preferences/constants";
+
+function appendPortMappings(summary: MarkdownString, container: ContainerInspectResponse): void {
+  const ports = container.NetworkSettings?.Ports ?? {};
+  const portMappings = Object.entries(ports).map(([containerPort, hostBindings]) => {
+    const hostPort = hostBindings?.[0]?.HostPort;
+    return { containerPort, hostPort: hostPort ?? containerPort };
+  });
+
+  if (portMappings.length) {
+    summary.appendMarkdown("\n\n### Ports");
+    portMappings.forEach(({ containerPort, hostPort }) => {
+      summary.appendMarkdown(`\n- ${containerPort} → ${hostPort}`);
+    });
+  }
+}
+
+function appendEnvironmentVars(summary: MarkdownString, container: ContainerInspectResponse): void {
+  const env: string[] = container.Config?.Env ?? [];
+  const relevantEnvVars = env.filter(
+    (envVar) =>
+      (envVar.startsWith("KAFKA_") || envVar.startsWith("SCHEMA_REGISTRY_")) &&
+      envVar.includes("="),
+  );
+
+  if (relevantEnvVars.length) {
+    summary.appendMarkdown("\n\n### Environment Variables");
+    relevantEnvVars.forEach((envVar) => {
+      const [key, value] = envVar.split("=");
+      if (value) {
+        summary.appendMarkdown(`\n- ${key}: ${value}`);
+      }
+    });
+  }
+}
 
 /**
  * Create a string representation of a {@link ContainerInspectResponse} object, limiting the
  * output to only the relevant Kafka/SR container information:
  * - Container name
+ * - Port mappings
  * - Image name (repo:tag)
  * - Environment variables that start with `KAFKA_`
  * - Status (running, exited, etc.)
  */
 export function summarizeLocalDockerContainer(container: ContainerInspectResponse): string {
   const containerName = container.Name?.replace("/", "") ?? "";
-  const summary = new MarkdownString().appendMarkdown(`### "${containerName}"`);
+  const summary = new MarkdownString().appendMarkdown(`# "${containerName}"`);
```

Comment:
```suggestion
  const summary = new MarkdownString().appendMarkdown(`## "${containerName}"`);
```
Very minor formatting change here, just to make sure the model can keep track of the header hierarchy (since the connection type+count uses `h1`):
<img width="867" alt="image" src="https://github.com/user-attachments/assets/e4894d73-d1f0-4e50-8a05-455ceab06ddd" />